services/ext/api.go

```diff
@@ -814,6 +814,10 @@ func (api *PublicAPI) MarkAllRead(ctx context.Context, chatID string) error {
 	return api.service.messenger.MarkAllRead(ctx, chatID)
 }
 
+func (api *PublicAPI) MarkCommunityNotificationRead(ctx context.Context, communityID string) error {
+	return api.service.messenger.MarkReadCommunityOverview(ctx, communityID)
+}
```

Comment:
why we have different names: `MarkCommunityNotificationRead` / `MarkReadCommunityOverview` / `DismissActivityCenterNotificationsCommunity` , i'd prefer `DismissActivityCenterNotificationsByCommunity` , pretty straight for me :)

Reply:
sorry @qfrank my bad. will fix names

Reply:
@qfrank i applied changes, could you please recheck . thanks